pages/common/gh-auth.md

```diff
@@ -0,0 +1,24 @@
+# gh-auth
+
+> Authenticate with GitHub in GitHub cli.
+> More information: <https://cli.github.com/manual/gh_auth_login>.
+
+- Login with interactive prompt:
+
+`gh auth login`
+
+- Login with token from https://github.com/settings/tokens :
+
+`echo {{your_token}} | gh auth login --with-token`
+
+- Check if you are logged in:
```

Comment:
```suggestion
- Verify your authentication state:
```

Reply:
From my point of view: if this should be beginner friendly TL;DR, I would keep it simple - since "login" and "*authentication state*" mean pretty much the same